Makefile

```diff
@@ -86,11 +101,7 @@ buildimage: build-systemutil
         --build-arg jenkins_build_number=${BUILD_NUMBER} \
         --build-arg REPO_SOURCE_URL=${REPO_SOURCE_URL} \
         --build-arg BUILD_URL=${BUILD_URL} \
-        --build-arg PROXY_IMAGE_URL=${PROXY_IMAGE_URL} \
-	-t $(IMAGE):$(VERSION)-$(ARCH) -f Dockerfile .
-ifeq ($(ARCH), amd64)
-	docker tag $(IMAGE):$(VERSION)-$(ARCH) $(IMAGE):$(VERSION)
-endif
+	-t $(CORE_DRIVER_IMG)-$(ARCH):$(TAG) -f Dockerfile .
```

Comment:
all other cloud provider using same format ?

Reply:
I see no standard format while checking other cloud providers. 
aws uses `IMG:<TAG> `
So, now changed the tagging as below syntax
We can retain image name as is and include ARCH info in tag
Example
```
gcr.io/k8s-staging-cloud-provider-ibm/ibm-vpc-block-csi-driver:ppc64le-dev
```